core/src/main/scala/org/apache/spark/deploy/master/Master.scala

```diff
@@ -757,7 +758,7 @@ private[spark] class Master(
       case fnf: FileNotFoundException =>
         // Event logging is enabled for this application, but no event logs are found
         val title = s"Application history not found (${app.id})"
-        var msg = s"No event logs found for application $appName in $eventLogFile."
+        var msg = s"No event logs found for application $appName in ${app.desc.eventLogDir}."
```

Comment:
great. This used to say something like "No event logs found for application X in /history/not-found"